lib/ansible/utils/module_docs_fragments/cnos.py

```diff
@@ -80,6 +80,6 @@ class ModuleDocFragment(object):
             - This specifies the type of device where the method is executed.
         required: Yes
         default: null
-        choices: [g8272_cnos,g8296_cnos,g8332_cnos]
-        version_added: 2.3
+        choices: [g8272_cnos,g8296_cnos,g8332_cnos,NE1072T_cnos,NE1032_cnos,NE1032T_cnos,NE10032_cnos,NE2572_cnos]
+        version_added: 2.4
```

Comment:
Changed it and created a new pull request. Please review.
https://github.com/ansible/ansible/pull/26780